components/header.tsx

```diff
@@ -32,6 +38,8 @@ const Header = () => {
         if (InnerRoutes.includes(pathname)) {
             // Use InnerHeader if route matches inner routes
             setHeaderType(HeaderTypes.INNER);
+        } else if (HeaderlessRoutes.includes(pathname)) {
+            setHeaderType(HeaderTypes.HEADERLESS);
```

Comment:
I'm going with the existing pattern here, but I recommend we include layout components on a per-page component rather than a global layout. It'll add a little duplication here and there, but will be better for extensibility. May tackle that cleanup in a future PR.